src/olympia/lib/akismet/tasks.py

```diff
@@ -16,7 +16,8 @@ def submit_to_akismet(report_ids, submit_spam):
 @task(ignore_result=False)
 def comment_check(report_ids):
```

Comment:
Could you rename this task ? It's a bit confusing with `AkismetReport.comment_check()`. Not new, but stumbled upon it because I was trying to see if other parts of the code needed changing.

Reply:
It was kinda deliberately named the same because it's a wrapper around `AkismetReport.comment_check` :)  But can do I guess.

Reply:
Yeah, I figured... Maybe just add `akismet` in the name just to differentiate it ? It's no big deal anyway, up to you.